src/cosmoz-tab-card.js

```diff
@@ -104,7 +104,7 @@ const style = css`
 		display: flex;
 		align-items: center;
 		gap: 8px;
-		background-color: #fff;
+		background-color: var(--cosmoz-tab-card-bg-color, white);
```

Comment:
I noticed that this is set on the `.header` element. The name `--cosmoz-tab-card-bg-color` suggests that it would affect the bg color of the entire tab card, not just the header. Are you sure this is correct?

Reply:
Oj, I do want this to affect the whole card. I have to change it also for the body

Reply:
Addressed